feature-registry.csv

```diff
@@ -0,0 +1,126 @@
+# New features should be added to the bottom to not disrupt existing numbering.
+tag,Friendly Name,Is Default?
+aalt,Access All Alternates,#N/A
+abvf,Above-base Forms,1
+abvm,Above-base Mark Positioning,1
+abvs,Above-base Substitutions,1
+afrc,Alternative Fractions,0
+akhn,Akhand,1
+blwf,Below-base Forms,1
+blwm,Below-base Mark Positioning,1
+blws,Below-base Substitutions,1
+calt,Contextual Alternates,1
+case,Case-Sensitive Forms,0
+ccmp,Glyph Composition / Decomposition,1
+cfar,Conjunct Form After Ro,1
+chws,Contextual Half-width Spacing,#N/A
+cjct,Conjunct Forms,1
+clig,Contextual Ligatures,1
+cpct,Centered CJK Punctuation,0
+cpsp,Capital Spacing,0
+cswh,Contextual Swash,0
+curs,Cursive Positioning,1
+cv01-cv99,Character Variants,0
+c2pc,Petite Capitals From Capitals,0
+c2sc,Small Capitals From Capitals,0
+dist,Distances,1
+dlig,Discretionary Ligatures,0
+dnom,Denominators,0

```

Comment:
HarfBuzz enabled `frac`, `numr`, and `dnom` automatically on numbers surrounding a fraction slash, so stripping these features by default will change rendering.